code/modules/antagonists/heretic/heretic_knowledge.dm

```diff
@@ -665,8 +665,13 @@
 	cost = 2
 	priority = MAX_KNOWLEDGE_PRIORITY + 1 // Yes, the final ritual should be ABOVE the max priority.
 	required_atoms = list(/mob/living/carbon/human = 3)
-	//use this to store the achievement typepath
+	/// The typepath of the achievement to grant upon successful ascension.
 	var/datum/award/achievement/misc/ascension_achievement
+	/// The text of the ascension announcement.
+	/// %NAME% is replaced with the heretic's real name.
+	var/announcement_text
+	/// The sound that's played for the ascension announcement.
+	var/announcement_sound
```

Comment:
The corresponding fake heretic ascension hallucination should be refactored to grab from these vars